packages/altair-app/src/scss/components/_alert.scss

```
@@ -98,3 +98,43 @@ nz-alert {
     color: var(--theme-font-color);
   }
 }
+
+// ---
+
+.altair-toast {
+  --normal-text: var(--theme-font-color);
+  --normal-bg: rgba(var(--rgb-theme-bg), 0.6);
+  // Default sonner styles
```

Stylelint (17.12.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)